### PR TITLE
Minor modifications to Night watch test and to the example templates

### PR DESCRIPTION
**Pull request by secretsayan**

### CODE REVIEW

File: example/.github/workflows/orca.yml [hunk 1]
```diff
@@ -60,13 +60,13 @@ jobs:
       # valid path to a YAML file relative to ORCA itself, e.g.,
       # ../example/tests/packages.yml.
       # @see https://github.com/acquia/orca/blob/develop/config/packages.yml
-      # ORCA_PACKAGES_CONFIG: ../example/tests/packages.yml
+      ORCA_PACKAGES_CONFIG: ../example/tests/packages.yml
```
Comment (TravisCarden) [inline example/.github/workflows/orca.yml:63]: This should stay commented out.

File: example/.github/workflows/orca.yml [hunk 2]
```diff
@@ -60,13 +60,13 @@ jobs:
...
       #
       # If your package isn't in ORCA's packages.yml or conflicts with details
       # that are, you can modify the active packages configuration at runtime by
       # uncommenting the following line and specifying your own alter file.
       # @see https://github.com/acquia/orca/blob/main/docs/advanced-usage.md#ORCA_PACKAGES_CONFIG_ALTER
-      # ORCA_PACKAGES_CONFIG_ALTER: ../example/tests/packages_alter.yml
+      ORCA_PACKAGES_CONFIG_ALTER: ../example/tests/packages_alter.yml
```
Comment (TravisCarden) [inline example/.github/workflows/orca.yml:69]: This should stay commented out.

File: example/.travis.yml [hunk 3]
```diff
@@ -74,13 +74,13 @@ env:
     # valid path to a YAML file relative to ORCA itself, e.g.,
     # ../example/tests/packages.yml.
     # @see https://github.com/acquia/orca/blob/develop/config/packages.yml
-    # - ORCA_PACKAGES_CONFIG: ../example/tests/packages.yml
+    - ORCA_PACKAGES_CONFIG=../example/tests/packages.yml
```
Comment (TravisCarden) [inline example/.travis.yml:77]: This should stay commented out. (Remember that this file is supposed to be copied into new projects as a template, and lines like this wouldn't be appropriate by default--they serve only as an example.)

File: example/.github/workflows/orca.yml [hunk 4]
```diff
@@ -93,6 +93,8 @@ jobs:
       # @see https://github.com/acquia/orca/blob/main/docs/advanced-usage.md#ORCA_AMPLITUDE_API_KEY
       # ORCA_TELEMETRY_ENABLE: "TRUE"
       #
+      # Enable Nightwatch.js testing. Acceptable values are "TRUE" and "FALSE".
```
Comment (TravisCarden) [inline example/.github/workflows/orca.yml:96]: This comment should reflect that fact that if you set this value to a literal boolean here `orca-enable-nightwatch` won't have any effect anymore on the individual jobs.

File: example/.travis.yml [hunk 5]
```diff
@@ -74,13 +74,13 @@ env:
...
     #
     # If your package isn't in ORCA's packages.yml or conflicts with details
     # that are, you can modify the active packages configuration at runtime by
     # uncommenting the following line and specifying your own alter file.
     # @see https://github.com/acquia/orca/blob/main/docs/advanced-usage.md#ORCA_PACKAGES_CONFIG_ALTER
-    # - ORCA_PACKAGES_CONFIG_ALTER=../example/tests/packages_alter.yml
+    - ORCA_PACKAGES_CONFIG_ALTER=../example/tests/packages_alter.yml
```
Comment (TravisCarden) [inline example/.travis.yml:83]: This should stay commented out.

File: example/.travis.yml [hunk 6]
```diff
@@ -102,13 +102,21 @@ env:
...
     #
     # To enable telemetry with Amplitude on cron runs, uncomment the following
     # line and set ORCA_AMPLITUDE_API_KEY in your Travis CI repository settings:
     # @see https://github.com/acquia/orca/blob/main/docs/advanced-usage.md#ORCA_TELEMETRY_ENABLE
     # @see https://github.com/acquia/orca/blob/main/docs/advanced-usage.md#ORCA_AMPLITUDE_API_KEY
     # - ORCA_TELEMETRY_ENABLE=TRUE
+    #
+    # Change the path where ORCA looks for the SUT. Accepted values are any valid directory path,
+    # e.g., /var/www/example, or ../example.
+    - ORCA_SUT_DIR=${TRAVIS_BUILD_DIR}/example
```
Comment (TravisCarden) [inline example/.travis.yml:115]: This line needs to be commented out.

File: example/.travis.yml [hunk 7]
```diff
@@ -102,13 +102,21 @@ env:
...
+    #
+    # Change the path where ORCA stores Clover file,
+    # e.g., logs/clover-self.xml
+    - ORCA_COVERAGE_CLOVER=$HOME/build/logs/clover-self.xml
```
Comment (TravisCarden) [inline example/.travis.yml:119]: This seems out of scope for this PR. Did you mean to add it?

**Review by TravisCarden — CHANGES_REQUESTED** [review]

File: example/.travis.yml (hunk 7 above)
Comment (secretsayan) [reply example/.travis.yml:119]: I will keep it commented

**Comment by secretsayan** [comment]
@TravisCarden Please review and let me know if any changes are required.
